messages/context.json

```diff
@@ -175,5 +175,10 @@
   "admin/editor.product-description.title": "Product description",
   "admin/editor.product-description.title-prop.title": "Title",
   "admin/editor.search-bar.title": "Search bar",
-  "admin/editor.search-bar.placeholder.title": "Placeholder"
+  "admin/editor.search-bar.placeholder.title": "Placeholder",
+  "store/editor.action-go-to": "Back to top",
```

Comment:
> `store/editor.action-go-to`

please rename this to `store/back-to-top.label`, here and on all instances where it appears

Reply:
@lbebber Done 😄